var/spack/repos/builtin/packages/rempi/package.py

```diff
@@ -20,3 +20,9 @@ class Rempi(AutotoolsPackage):
     depends_on("automake", type='build')
     depends_on("libtool", type='build')
     depends_on("libpciaccess", type='link')
+
+    def setup_environment(self, spack_env, run_env):
```

Comment:
This method is legacy/deprecated.  Do you mean to use `setup_build_environment(self, env)` here?